source/indexes.txt

```diff
@@ -116,6 +116,39 @@ input.
    |compass-short| shows the type of your new index as
    :guilabel:`Wildcard`.
 
+.. _hide-index:
+
+Hide an Index
+-------------
+
+You can :manual:`hide an index </core/index-hidden>` from the query planner to 
+evaluate the potential impact of dropping an index without actually dropping the 
+index.
+
+.. procedure:: 
+   :style: normal 
+   
+   .. step:: Hover over the index. 
+
+      From the :guilabel:`Indexes` tab, hover over the index you want to hide.
+
+   .. step:: Click the :guilabel:`Hide Index` button.
+
+      Click the closed-eye icon on the right that appears when you hover over 
+      your selected index.
+
+   .. step:: Click :guilabel:`Confirm`.
+      
+      In the dialog box, confirm the index you want to hide. After you confirm 
+      your selection, a :guilabel:`Hidden` badge appears under the 
+      :guilabel:`Properties` column.
+
+   .. step:: (Optional) Unhide your index. 
```

Comment:
[formatting suggestion]

Maybe rather than making this a step in the Hide an Index procedure, we can make this its own paragraph after step 3. This might be better because this step isn't really part of the steps to Hide an Index.

If we make this change, consider updating the section heading to "Hide or Unhide an Index".

Reply:
Ahh gotcha. Thanks for that suggestion! Makes more sense.